pwd_guess.py

```diff
@@ -826,6 +830,13 @@ def __init__(self, pwd_list, config=ModelDefaults(), multi_gpu=1):
         self.model_to_save = None
         self.multi_gpu = multi_gpu
         self.pwd_list = pwd_list
+        self.cumulative_chunks = 0
+        self.min_loss_early_stopping = 999999999 #A really large number
+        self.poor_batches_early_stopping = 0
+        if config.tensorboard:
+            self.callback = TensorBoard(".")
```

Comment:
Can we make the path to put the tensorboard logs configurable either with a Config parameter or a command line argument?